test/unit/services/EmailServiceSpec.scala

```diff
@@ -45,13 +45,13 @@ class EmailServiceSpec extends BaseSpec {
   private val gyeRecipientFullName = "John Doe"
   private val gyeOrgName = "Test Company Name"
   private val gyeCompletionDate = "5 May 2017"
-  private val gyeRecipientDetails = RecipientDetails("GetYourEORI", gyeRecipientEmailAddress, gyeRecipientFullName, Some(gyeOrgName), Some(gyeCompletionDate))
+  private val gyeRecipientDetails = RecipientDetails("Register", "ATar", gyeRecipientEmailAddress, gyeRecipientFullName, Some(gyeOrgName), Some(gyeCompletionDate))
 
   private val migrateRecipientEmailAddress = "jane.doe@example.com"
   private val migrateRecipientFullName = "Jane Doe"
   private val migrateOrgName = "Test Company Name 2"
   private val migrateCompletionDate = "23 June 2004"
-  private val migrateRecipientDetails = RecipientDetails("Migrate", migrateRecipientEmailAddress, migrateRecipientFullName, Some(migrateOrgName), Some(migrateCompletionDate))
+  private val migrateRecipientDetails = RecipientDetails("Subscribe", "ATar", migrateRecipientEmailAddress, migrateRecipientFullName, Some(migrateOrgName), Some(migrateCompletionDate))
```

Comment:
And similar too, I would use `ATaR`
The same comment line 48

Reply:
I looked up "ATar" on one of the architect documents.   But I'll change it if you think it's wrong